frontend/app/layout.tsx

```diff
@@ -5,6 +5,7 @@ import { ThemeProvider } from "@/components/theme-provider";
 import { ThemeToggle } from "@/components/theme-toggle";
 import { Toaster } from "@/components/ui/toaster";
 import AuthProvider from "./auth/auth-context";
+import { Toaster } from "@/components/ui/toaster";
```

Comment:
There seems to be a duplicate `Toaster` import statement

Reply:
Removed